app.js

```diff
@@ -1,21 +1,29 @@
 const express = require('express');
+require("dotenv").config();
```

Comment:
dotenv requirement looks good.
Maybe we can have it on the first line to explicitly indicate that it runs before any other requirement.

Reply:
Eagle eyes!
thanks @twumm